src/pages/index.js

```diff
@@ -1,57 +1,107 @@
 import FormValidator from "../components/FormValidator.js";
-import { initialCards, settings } from "../utils/constants.js";
+import { settings } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import "./index.css";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithForm from "../components/PopupWithForm.js";
-("../components/PopupWithForm.js");
+import Api from "../components/Api.js";
+import PopupConfirmation from "../components/PopupConfirmation.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "967eb4d1-2e88-4a75-bdf4-9be2dee67768",
+    "Content-Type": "application/json",
+  },
+});
 
-/* Elements */
+api.getProfileInfo().then(({ name, about, avatar }) => {
+  userInfo.setUserInfo({ name, about });
+  userInfo.setAvatar({ avatar });
+});
 
+api.getInitialCards().then((cardData) => {
+  const section = new Section({ items: cardData, renderer }, ".cards__list");
```

Comment:
Only **one instance** of `Section` should be created in the project because there is only **one** section for the cards.

Reply:
when I try to remove the duplicate Section from my getInitialCards everything disappears, how do I resolve this?

Reply:
> when I try to remove the duplicate Section from my getInitialCards everything disappears, how do I resolve this?

You can change `renderItems` so that it receives the array of cards


```js
section.renderItems(cardData);

```

Reply:
Even after changing my renderitems to receive cardData my cards disappear